lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -521,6 +521,7 @@ def test_get_threads_by_topic_id(self):
         self.get_thread_list([], topic_id_list=["topic_x", "topic_meow"])
         self.assertEqual(urlparse(httpretty.last_request().path).path, "/api/v1/threads")
         self.assert_last_query_params({
+            "user_id": [unicode(self.user.id)],
```

Comment:
assert_last_query_params doesn't do subset matching, and the lower layers of code add in the user_id parameter. The CS uses it for various things like updating read states or determining which user is flagging a thread. In this case it isn't necessary, but it's easier to avoid changing the lower layers.